README.md

```diff
@@ -16,41 +16,118 @@ The purpose of the framework is to provide a simple in app solution for player c
+## Communication
 
-5. Skip a certain time interval using method:
-*		public func skipDirection(skipDirection: iOSAudioPlayer.SkipDirection, timeInterval: Double, offset: Double)
+- If you **found a bug**, open an issue [github](https://github.com/3pillarlabs/ios-audio-player/issues/new).
+- If you **have a feature request**, open an issue [github](https://github.com/3pillarlabs/ios-audio-player/issues/new).
+- If you **want to contribute**, submit a pull request [github](https://github.com/3pillarlabs/ios-audio-player/compare).
 
-6. Seek the player to a certain offset:
-*		public func seekPlayerToTime(value: Double, completion: (() -> Void)!)
+## Installation
+### CocoaPods
 
-7. Check the duration of the current player item:
-*		public var durationInSeconds: Double { get }
+[CocoaPods](http://cocoapods.org) is a dependency manager for Cocoa projects and is the preferred method of installation.
 
-8. Check current progress of the player on the current player item:
-*    	public var currentTimeInSeconds: Double { get }
+Install the latest version of CocoaPods with the following command:
 
-9. Listen to the notification which is triggered when a certain media file package is loaded:
-* 		public let TPGMediaLoadedStateNotification: String
+```bash
+$ gem install cocoapods
+```
 
-In the demo project you're able to see how the framework is used.        
+To integrate iOSAudioPlayer into your Xcode project that does not have CocoaPods integrated, you need to run in your project's folder the command:
+```bash
+$ pod init
+```
+This will create a new file called `Podfile`.
 
-## License
+After this you need to add a new entry in your target's pods specifications.
+```ruby
+pod 'iOSAudioPlayer'
+```
+Once this is done, in your `Podfile` you should have something like this:
+```ruby
+platform :ios, '8.0'
+use_frameworks!
+
+target 'yourTargetName' do
+    pod 'iOSAudioPlayer'
+end
+```
+
+To integrate iOSAudioPlayer into your Xcode project that already has CocoaPods integrated, just add in your `Podfile`:
+```ruby
+pod 'iOSAudioPlayer'
+```
+Once this is done, in your `Podfile` you should have something like this:
+```ruby
+platform :ios, '8.0'
+use_frameworks!
+
+target 'yourTargetName' do
+    // other pods
+    pod 'iOSAudioPlayer'
+end
+```
+
+Then, run the following command:
+```bash
+$ pod update
+```
+
+Inside your code, in the file where you want to use the iOSAudioPlayer classes, add the following import statement:
+```swift
+import iOSAudioPlayer
+```
+
+### Manually
+
+If you prefer not to use CocoaPods, you can integrate iOSAudioPlayer into your project manually.
+
+#### Source File
+
+If you prefer to rock it old-school, iOSAudioPlayer can be integrated by adding all the Swift files located inside the `Classes` directory (`Classes/*.*`) directly into your project. Note that you will no longer need to `import iOSAudioPlayer` since you are not actually loading a framework.
```

Comment:
Proposing the user to not use our repository will help him when we'll be updating the framework because he'll need to copy the files again and again and again...
Instead I propose to encourage him to use git submodule. See [this](https://github.com/Alamofire/Alamofire#manually) example.